src/onegov/form/models/definition.py

```diff
@@ -214,3 +221,176 @@ def for_new_name(self, name: str) -> 'Self':
             payment_method=self.payment_method,
             created=self.created
         )
+
+
+class SurveyDefinition(Base, ContentMixin, TimestampMixin,
+                       Extendable):
+    """ Defines a survey stored in the database. """
+
+    __tablename__ = 'surveys'
+
+    # # for better compatibility with generic code that expects an id
+    # # this is just an alias for `name`, which is our primary key
+    @hybrid_property
+    def id(self) -> str:
+        return self.name
+
+    #: the name of the form (key, part of the url)
+    name: 'Column[str]' = Column(Text, nullable=False, primary_key=True)
```

Comment:
Rather than copying form definitions here I'd suggest specifying a standard `id` field with a UUID or integer. The fewer special cases like `FormDefinition` we have, the better. You can keep the name as a URL key by making it `unique=True` instead of `primary_key=True`.

Reply:
It may also be worth considering putting these all in the same table with the same abstract polymorphic base-class. So when we add new features to forms, they will copy over to surveys without needing to duplicate all the code. Especially because the `RegistrationWindow` and `FormSubmission` probably don't need their own subclass for surveys, any changes in logic can be made based on the type of the linked definition. If it's all in the same table we don't need to duplicate all the related tables.